client/state/jetpack-connect/reducer/schema.js

```diff
@@ -1,26 +1,31 @@
 /** @format */
 export const jetpackConnectAuthorizeSchema = {
-	type: 'object',
-	additionalProperties: false,
-	patternProperties: {
-		'^.+$': {
+	anyOf: [
+		{
+			type: 'object',
+			additionalProperties: false,
+			properties: {},
+		},
+		{
 			type: 'object',
+			additionalProperties: false,
 			required: [ 'timestamp' ],
 			properties: {
-				authorizationCode: { type: 'string ' },
```

Comment:
See that ugly space in there? New unit tests caught this one 😎

Reply:
🎉